airbyte-integrations/connectors/source-zendesk-talk/setup.py

```diff
@@ -0,0 +1,49 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
+    "airbyte-protocol",
+    "base-python",
+    "backoff==1.10.0",
+    "pendulum==1.2.0",
+    "requests==2.25.1",
+]
+
+TEST_REQUIREMENTS = ["pytest", "requests_mock==1.8.0"]
+
+
+setup(
+    name="source_zendesk_talk",
+    description="Source implementation for Zendesk Talk.",
+    author="Airbyte",
+    author_email="contact@airbyte.io",
+    packages=find_packages(),
+    install_requires=MAIN_REQUIREMENTS,
+    package_data={"": ["*.json", "schemas/*.json"]},
+    extras_require={
```

Comment:
can this go in install_requires? It's simpler for local dev iteration. Otherwise someone iterating would need to know to run `pip install ".[test]"

Reply:
If I understood you correctly, I removed the `extras_require` field and expanded `install_requires` to:
`install_requires=MAIN_REQUIREMENTS + TEST_REQUIREMENTS`